### PR TITLE
Fix bot.time.day and add more values

### PR DESCRIPTION
**Pull request by Naomi-alt**
I'm not sure whether bot.time.time is the best name.
And also someone else needs to update the russian api doc.

### CODE REVIEW

File: lib/plugins/bed.js [hunk 1]
```diff
@@ -84,7 +84,7 @@ function inject (bot) {
 
   function sleep (bedBlock, cb) {
     cb = cb || noop
-    if (!(bot.time.day >= 12541 && bot.time.day <= 23458)) {
+    if (!(bot.time.timeOfDay >= 12541 && bot.time.timeOfDay <= 23458)) {
```
Comment (Karang) [inline lib/plugins/bed.js:87]: What about defining isDay to timeOfDay out of [12542, 23458] so it can be used here ?

Comment (Naomi-alt) [reply lib/plugins/bed.js:87]: I guess that just depends on when you consider it to be night.
That definition would be the time at which you can sleep.
I used the moment mobs are able to spawn.
https://minecraft.gamepedia.com/Daylight_cycle

Comment (Karang) [reply lib/plugins/bed.js:87]: ok, fair enough.

**Comment by rom1504** [comment]
Kind of breaking but let's count it as a bug fix, the previous thing was weird, I doubt many people used it

**Pull request merged**